src/models/DAO.js

```diff
@@ -31,39 +31,66 @@ class Events extends BaseEvents {
 
 const listen = async (dao) => {
   const key = toKey(dao.id, 'SerializedListener');
+
   if (cache[key]) {
     return;
   }
-  const listenerSubject = await dao.events.Serialized();
-  listenerSubject.subscribe(dao.refresh.bind(dao));
-  cache[key] = true;
+
+  try {
+    cache[key] = true;
```

Comment:
Any reason to include the boolean assignment inside of the try block?

Reply:
It seemed cleaner to me this way. Other than that, no.